embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/testcases/V1SessionApiTest.kt

```diff
@@ -0,0 +1,38 @@
+package io.embrace.android.embracesdk.testcases
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import io.embrace.android.embracesdk.Embrace
+import io.embrace.android.embracesdk.IntegrationTestRule
+import io.embrace.android.embracesdk.ResourceReader
+import io.embrace.android.embracesdk.recordSession
+import io.embrace.android.embracesdk.validatePayloadAgainstGoldenFile
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class V1SessionApiTest {
+
+    @Rule
+    @JvmField
+    val testRule: IntegrationTestRule = IntegrationTestRule()
+
+    /**
+     * Verifies that a session end message is sent.
+     */
+    @Test
+    fun sessionEndMessageTest() {
+        with(testRule) {
+            val message = harness.recordSession {
+                embrace.setUserIdentifier("some id")
+                embrace.setUserEmail("user@email.com")
+                embrace.setUsername("John Doe")
+
+                // add webview information
+                val msg = ResourceReader.readResourceAsText("expected-webview-core-vital.json")
```

Comment:
`expected-webview-core-vital.json` is the `expected` in this naming correct? AFAIK this is not what we expect, but what we inject instead, right?

Reply:
Yep - this is what we inject